canonical-kubernetes/steps/04_enable-cni/after-deploy

```diff
@@ -4,12 +4,14 @@ set -eu
 
 . "$CONJURE_UP_SPELLSDIR/sdk/common.sh"
 
-cni_implementation="$(dirname "$0")/$JUJU_PROVIDERTYPE/enable-cni"
-if [[ -f "$cni_implementation" ]]; then
-    . "$cni_implementation"
-    enable_cni
-    setResult "Cloud native integration enabled"
-else
-    setResult "Cloud native integration not available"
-fi
-exit 0
+case "$(getStepKey "cni-enabled")" in
+    true)
+        setResult "Cloud native integration enabled"
+        ;;
+    false)
```

Comment:
I see `before-wait` sets `cni-enabled` to `true || unavailable`.  Is this `false` case for a future impl?

Reply:
Yeah, there's no path that will hit this now, since most of the heavy lifting is handled by the charms, but I just decided I'd keep it here in case there were some future cases where we wanted to set up integration but were prevented for some reason outside of the work done by the charms.  Maybe something like juju trust is available but fails, in which case we probably don't want to fall back to setting the config.